sunpy/map/mapsequence.py

```diff
@@ -271,6 +271,8 @@ def plot(self, axes=None, resample=None, annotate=True,
         plot_function : function
             A function to be called as each map is plotted.
             For more information see `sunpy.visualization.animator.MapSequenceAnimator`.
+        norm : can be explicitly used to call mapsequence.plot function
+        cmap : can be explicitly used to call mapsequence.plot function
```

Comment:
```suggestion
        norm : 
            Normalization used in scaling the plot.
            This will override the predefined value in ``plot_settings['norm']``.
        cmap : 
            Color map to be used in coloring the plot.
            This will override the predefined value in ``plot_settings['cmap']``.
```
The types are missing for both, I don't know what they are off by hand, you will need to find them either via Matplotlib's documentation or get into the code and check their type.